tf-modules/archive/main.tf

```diff
@@ -10,3 +10,9 @@ locals {
     var.elasticsearch_security_group_id
   ])
 }
+
+data "aws_lambda_invocation" "verify_provider_secrets_migration" {
```

Comment:
So this activates the verification lambda.... but that lambda has the following in it: 


```
 else {
    if (isNil(provider.username) && isNil(provider.password)) return;
    throw new Error(`Provider ${provider.id} has plaintext username or password. Must invoke the providerSecretsMigration Lambda function.`);
  }
```

We should probably update that logic, given we're removing the lambda and that should no longer be possible.